### PR TITLE
tests: upgrade devtools web tests deps, use python3

### PR DESCRIPTION
**Pull request by connorjclark**
blink tests were converted to p3 just a few weeks ago

### CODE REVIEW

File: lighthouse-core/test/chromium-web-tests/README.md [hunk 1]
```diff
@@ -21,15 +21,12 @@ SKIP_DOWNLOADS=1 yarn test-devtools
...
-  Ensure you have `requests` module available globally on your python 2.7 install. New Macs do not come with pip for python 2.7 which is deprecated, so you might have to install that too.
+  <summary>Install python dependencies</summary>
 
+  Ensure you have these modules available globally on your python 3.x install:
 
   ```sh
-  curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
-  python get-pip.py
-  pip -m pip install requests
+  python3 -m pip install requests six
```
Comment (adamraine) [inline lighthouse-core/test/chromium-web-tests/README.md:29]: Do we need to use this command in CI now?

Comment (connorjclark) [reply lighthouse-core/test/chromium-web-tests/README.md:29]: This isn't new, just wasn't documented. CI still fine

Comment (adamraine) [reply lighthouse-core/test/chromium-web-tests/README.md:29]: I mean, CI is still complaining about the missing `six` module. This was my best guess as to why.

Comment (connorjclark) [reply lighthouse-core/test/chromium-web-tests/README.md:29]: Oh weird I was just looking at it said it passed. Gh lying to me

**Review by adamraine — APPROVED** [review]

**Pull request merged**